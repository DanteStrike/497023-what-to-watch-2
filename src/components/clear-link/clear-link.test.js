import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter as Router} from "react-router-dom";
import ClearLink from "./clear-link.jsx";

it(`Component ClearLink should render correctly`, () => {
  const MockComponent = () => <div/>;

  const component = renderer.create(
      <Router>
        <ClearLink to="any">
          <MockComponent/>
        </ClearLink>
      </Router>
  ).toJSON();

  expect(component).toMatchSnapshot();
});
