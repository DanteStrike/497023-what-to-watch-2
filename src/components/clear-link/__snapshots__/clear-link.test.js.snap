// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ClearLink should render correctly 1`] = `
<a
  href="/any"
  onClick={[Function]}
  style={
    Object {
      "color": "unset",
      "textDecoration": "none",
    }
  }
>
  <div />
</a>
`;
